src/Input/KeyInterop.ts

```diff
@@ -170,8 +170,9 @@ module Fayde.Input {
             }
             return args;
         }
+		
         CreateArgsDown(e): Fayde.Input.KeyEventArgs {
-            if (e["char"] && e.keyCode !== 8 && e.keyCode !== 9)
+            if (e["char"] && e.keyCode !== 8 && e.keyCode !== 9 && !e.ctrlKey)

```

Comment:
These worry me.  Why do we want keys that are normally filtered out to pass through while ctrl key is held.